x/qgb/types/genesis.go

```diff
@@ -61,6 +63,12 @@ func validateDataCommitmentWindow(i interface{}) error {
 			val,
 			MinimumDataCommitmentWindow,
 		))
+	} else if val > uint64(appconsts.DataCommitmentBlocksLimit) {
```

Comment:
not super related to this PR, but we don't need "else if" here, we can just have an if statement since we're returning

Reply:
[bebbbf9](https://github.com/celestiaorg/celestia-app/pull/1803/commits/bebbbf9b1343d52d2300b885c26465f633e2617d)